rskj-core/src/main/java/co/rsk/peg/ErpFederation.java

```diff
@@ -45,7 +59,22 @@ public long getActivationDelay() {
         return activationDelay;
     }
 
-    public abstract Script getStandardRedeemScript();
+    public Script getStandardRedeemScript() {
```

Comment:
Should we start calling this the DefaultRedeemScript?

Reply:
yes. i think so